core/src/price_reporter/manager.rs

```diff
@@ -330,46 +245,4 @@ impl PriceReporterManagerExecutor {
         channel.send(price_reporter.peek_all_exchanges()).unwrap();
         Ok(())
     }
-
-    /// Handler for CreateNewMedianReceiver job.
-    fn create_new_median_receiver(
```

Comment:
why do we no longer support the `CreateNewMedianReceiver`, `GetSupportedExchanges`, and `GetHealthyExchanges` jobs?

Reply:
They were unused, we can add them back if we need them, but better to keep the interface as lean as possible